### PR TITLE
Mobile fixes

### PR DESCRIPTION
**Pull request by MeenuKumari28**
Fixed mobile issue and added width and padding to icon button.

### CODE REVIEW

**Comment by amitu** [comment]
@MeenuKumari28 delete the _Fifthtry folder please. Also resolve conflict.

**Review by Arpita-Jaiswal — APPROVED** [review]

File: index.ftd [hunk 1]
```diff
@@ -25,13 +21,16 @@ export: hero-with-two-cta
...
+export: heading-hero, heading-large, heading-medium, heading-small, heading-tiny,button-large, button-medium, button-small, copy-large, copy-regular, copy-small, fine-print, source-code, link
 
 -- ftd.color-scheme colors: $dark-flame-cs.main
 -- ftd.type-data types: $roboto-typography.types
 
 
+-- ftd.column:
+width: fill-container
+height: fill-container
+background.solid: $colors.background.base
```
Comment (Arpita-Jaiswal) [inline index.ftd:33]: Use `blank-page` only

File: typography.ftd [hunk 2]
```diff
@@ -1,19 +1,16 @@
 -- component heading-hero:
-module design-system: ds
-caption text:
+caption or body text:
```
Comment (Arpita-Jaiswal) [inline typography.ftd:2]:
```suggestion
caption text:
```
Heading components will only take values in caption

File: typography.ftd [hunk 3]
```diff
@@ -24,47 +21,76 @@ role: $inherited.types.heading-large
...
+ftd.color color: $inherited.colors.text-strong
+ftd.display display: inline
+
+-- ftd.text: $button-large.text
+color: $button-large.color
+role: $inherited.types.button-large
+
+-- end: button-large
+
```
Comment (Arpita-Jaiswal) [inline typography.ftd:66]: Button text can never be added independent of button, so remove all such components

File: typography.ftd [hunk 4]
```diff
@@ -24,47 +21,76 @@ role: $inherited.types.heading-large
...
+color: $button-small.color
+role: $inherited.types.button-small
+
+-- end: button-small
+
+
 -- component copy-large:
-module design-system: ds
-body text:
+caption or body text:
```
Comment (Arpita-Jaiswal) [inline typography.ftd:93]:
```suggestion
body text:
```

Copy large will not take text in caption area.

File: typography.ftd [hunk 5]
```diff
@@ -117,8 +140,21 @@ role: $inherited.types.fine-print
...
+ftd.color color: $inherited.colors.text
+ftd.display display: inline
+
+-- ftd.text: $source-code.text
+color: $source-code.color
+role: $inherited.types.source-code
+display: $source-code.display
+
+-- end: source-code
```
Comment (Arpita-Jaiswal) [inline typography.ftd:153]: Source code can be part of the code block only. It can't be an independent text component. So remove this.

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

**Comment by MeenuKumari28** [comment]
Mobile view fixes were merged.

**Pull request closed without merging**